library/core/src/str/mod.rs

```diff
@@ -1202,6 +1205,7 @@ impl str {
     ///
     /// [`split_whitespace`]: str::split_whitespace
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[doc(alias = "strtok")]
```

Comment:
This one is iffy, but C doesn't really have a split function and I didn't know where else to put strtok.